### DISCUSSION TITLE
Uzbek translation

### DISCUSSION THREAD

**Discussion by elzodxon** [vuejs-translations/guidelines#32] [General]
Let's translate into Uzbek language

**Comment by vayvoo** [comment]
how?

**Reply by dilshodturab** [reply]
Bugun nima qilishni bilmasdan oss ga contribute qilish anchadan beri qilaman - qilaman deb yurgandim. Izlab izlab topdim boshlanishiga qayerga borish kerakligini?

**Comment by Zikoi5** [comment]
https://vuejs.uz

Tarjimachi birodarlarni telegram guruhga taklif qilaman https://t.me/vuejs_uz_tarjima

Asosiy guruh https://t.me/vuejs_uz

https://github.com/Zikoi5/docs-uz

**Comment by president-tuychiyev** [comment]
Barchaga Assalomu alaykum! @Zikoi5  vuejs doc tarjimasi davom etyabdimi yoki yakunlanganmi ? vuejs.org tasdiqlamagnaligi sababli contributelar bilan gaplashgan edim doc tarjimasi to'liq bo'lmaganligi sababli tasdiqlanmagan ekan

**Comment by president-tuychiyev** [comment]
@Jinjiang Uzbek (uz) translation is 99% complete.

You can review it here: https://github.com/president-tuychiyev/docs-uz/tree/translated-to-uz

Could you help us proceed with making it official and transferring the repository to the vuejs-translations organization?

The project is ready for review and deployment. Thank you!

**Reply by Jinjiang** [reply]
Hey @president-tuychiyev very great job! I've sent you the invitation to join the Uzbek translation team. After accepting that, you are free to transfer the repo following our guidelines. Thanks!

**Reply by president-tuychiyev** [reply]
@Jinjiang Thank you! I've accepted the invitation and transferred the repository following the guidelines.

Please let me know if anything else is needed. Looking forward to contributing further.

**Reply by Jinjiang** [reply]
Great. I've found the repo is still forked from the docs repo which needs to be detached manually:

> Before performing the transfer, use the GitHub support request to [detach your fork](https://support.github.com/request/fork). This is necessary because GitHub doesn't allow multiple forks of the same source repo to co-exist in the same organization.

Please ensure it well detached.

At the same time, I will try to deploy it onto our official domain. I will let you know once it's done.

**Reply by president-tuychiyev** [reply]
Hi @Jinjiang 👋

The fork has been successfully detached, and the repository is now standalone.

Everything should be ready for deployment on your side. Let me know if anything else is needed.

Thanks!

**Reply by Jinjiang** [reply]
Hey, @president-tuychiyev sorry to reaching out to you late. When applying official domains, I had some discussions with teammates. We came to an agreement that generally AI translation quality is now high enough and more acceptable rather than long-term manual maintenance nowadays. Based on this point, we are going to not accept new translations. Existing translations which are not actively maintained will also be removed and redirected to the English docs.
More information: https://github.com/vuejs-translations/guidelines#why-we-are-making-this-change
However, we really appreciate the your effort on this. Hope you can understand that. 🙏

**Reply by president-tuychiyev** [reply]
> Hey, @president-tuychiyev sorry to reaching out to you late. When applying official domains, I had some discussions with teammates. We came to an agreement that generally AI translation quality is now high enough and more acceptable rather than long-term manual maintenance nowadays. Based on this point, we are going to not accept new translations. Existing translations which are not actively maintained will also be removed and redirected to the English docs. More information: https://github.com/vuejs-translations/guidelines#why-we-are-making-this-change However, we really appreciate the your effort on this. Hope you can understand that. 🙏

Hi, thank you for the clarification and the detailed explanation.

I understand the decision and the direction you're taking. Thanks for taking the time to provide clear reasoning and guidance.

I appreciate the opportunity to contribute, and I hope I’ll still be able to support the growth of Vue.js ecosystem in other ways going forward.

Thanks again 🙏

**Reply by Zikoi5** [reply]
https://github.com/president-tuychiyev/docs-uz/tree/translated-to-uz ni manda ochmayapti, 404
